widget_bandsplot/bandsplot.py

```diff
@@ -109,8 +110,11 @@ def __init__(
             self.dos_fermienergy = dos["fermi_energy"]
             temp_dos = deepcopy(dos)
 
+            ymin = []
+            ymax = []
+
             # Convert data to the format for which chart.js can plot directly
-            for i, d_dos in enumerate(temp_dos["dos"]):
+            for _i, d_dos in enumerate(temp_dos["dos"]):
```

Comment:
If _i not used, we don't need call enumerate to get the index of list element.

Reply:
Yes, I rmeoved it.